huggingface/pytorch/training/docker/1.11/py3/cu113/Dockerfile.trcomp.gpu

```diff
@@ -0,0 +1,109 @@
+# https://github.com/aws/deep-learning-containers/blob/master/available_images.md 
+# refer to the above page to pull latest Pytorch image
+
+# docker image region us-west-2
+FROM 763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-training:1.11.0-gpu-py38-cu113-ubuntu20.04-sagemaker
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# version args
+ARG TRANSFORMERS_VERSION=4.19.2
+ARG DATASETS_VERSION=1.18.4
+ARG PYTHON=python3
+
+ARG PT_BUCKET=https://sm-training-comp-pytorch-binaries.s3.us-west-2.amazonaws.com/617bcafe-7da3-4700-a4ba-6707a11c6630/20220526-213653/9ad1460a3411c47cc696c97b52d11dd3984fd2b1
+ARG PT_URL=${PT_BUCKET}/torch-1.11.0%2Bcu113-cp38-cp38-linux_x86_64.whl
+ARG PT_XLA_URL=${PT_BUCKET}/torch_xla-1.11.0-cp38-cp38-linux_x86_64.whl
+ARG TORCHVISION_URL=${PT_BUCKET}/torchvision-0.12.0a0%2B9b5a3fe-cp38-cp38-linux_x86_64.whl
+ARG TORCHAUDIO_URL=${PT_BUCKET}/torchaudio-0.11.0%2B6297e97-cp38-cp38-linux_x86_64.whl
+ARG HF_TRANSFORMERS_URL=https://sm-training-comp-pytorch-binaries.s3.us-west-2.amazonaws.com/d590055d-bfc4-439e-8199-2b51a9f46beb/20220521-013053/5906ed2b65214eddbb7ee27f08dbe1cad3ed1921/transformers-4.19.2-py3-none-any.whl
```

Comment:
[PT 1.11 Base DLC ](https://github.com/aws/deep-learning-containers/blob/master/pytorch/training/docker/1.11/py3/cu115/Dockerfile.gpu#L11)shipped with [TorchData](https://pytorch.org/data/beta/index.html). Please check if we need to rebuild this with trcomp support.

Reply:
I checked , we dont need to rebuild this.